src/app/containers/Navigation/index.amp.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Navigation from '@bbc/psammead-navigation';
+import { node, string, shape } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { ScrollableNavigation } from '@bbc/psammead-navigation/scrollable';
+import { AmpDropdown, AmpMenuButton } from '@bbc/psammead-navigation/dropdown';
+import { GEL_GROUP_2_SCREEN_WIDTH_MAX } from '@bbc/gel-foundations/breakpoints';
+import styled from 'styled-components';
+
+const DROPDOWN_ID = 'dropdown-menu';
+const NAVIGATION_ID = 'navigation-id';
+const SCROLLABLE_ID = 'scrollable-nav';
+const HIDDEN_CLASS_NAME = 'scrollable-hidden';
+const OPEN_CLASS_NAME = 'open';
+
+const StyledAmpScrollableNavigation = styled(ScrollableNavigation)`
+  &.${HIDDEN_CLASS_NAME} {
+    @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+      display: none;
+      visibility: hidden;
+    }
+  }
+`;
+
+const AmpNavigationContainer = ({
+  script,
+  service,
+  dir,
+  menuAnnouncedText,
+  scrollableListItems,
+  dropdownListItems,
+}) => (
+  <Navigation
+    script={script}
+    service={service}
+    dir={dir}
+    id={NAVIGATION_ID}
+    ampOpenClass={OPEN_CLASS_NAME}
+  >
+    <AmpMenuButton
+      announcedText={menuAnnouncedText}
+      onToggle={`
+        ${DROPDOWN_ID}.toggleVisibility,
+        ${SCROLLABLE_ID}.toggleClass(class=${HIDDEN_CLASS_NAME}),
+        ${NAVIGATION_ID}.toggleClass(class=${OPEN_CLASS_NAME})
+      `}
+      dir={dir}
+      script={script}
+    />
+    {/* Hidden attribute allows us to toggle visibility on the dropdown
+    using AMP actions. */}
+    <AmpDropdown id={DROPDOWN_ID} hidden>
+      {dropdownListItems}
+    </AmpDropdown>
+    <StyledAmpScrollableNavigation dir={dir} id={SCROLLABLE_ID}>
+      {scrollableListItems}
+    </StyledAmpScrollableNavigation>
```

Comment:
Probably fine, but in Canonical we have `Scrollable` then `Dropdown` in the DOM, here we have `Dropdown` then `Scrollable`.

It probably should be consistent across the two? Although only one will ever be visible at a time (especially in canonical where only one will be rendered at a time).

Reply:
I think I'll raise a ticket for this rather than fix it in this PR - would want to do a few a11y checks to make sure we haven't broken anything that passed the a11y swarm